### PR TITLE
Time to First Token Graph

### PR DESCRIPTION
**Pull request by maamalama**
Added Time to First Token Graph

### CODE REVIEW

**Review by chitalian — APPROVED** [review]
LGTM great work

File: web/lib/api/metrics/averageTimeToFirstToken.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    FROM request_response_log
+    WHERE (
+      (${filterString})
+    )
+  )
+  SELECT CASE
+    WHEN count = 0 THEN 0
+    ELSE total_time_to_first_token / count
+  END as average_time_to_first_token
+  FROM total_count
```
Comment (chitalian) [inline web/lib/api/metrics/averageTimeToFirstToken.ts:42]: Can we just select the average here? 

This query is resulting in some weird calcs:


image

Comment (chitalian) [reply web/lib/api/metrics/averageTimeToFirstToken.ts:42]: SELECT avg (request_response_log.time_to_first_token)

Where request_response_log.time_to_first_token > 0

Comment (chitalian) [reply web/lib/api/metrics/averageTimeToFirstToken.ts:42]: something like that ^

**Review by chitalian — CHANGES_REQUESTED** [review]

File: web/lib/api/metrics/averageTimeToFirstToken.ts (hunk 1 above)
Comment (maamalama) [reply web/lib/api/metrics/averageTimeToFirstToken.ts:42]: @chitalian , It's query for metric value (not overtime). I've fixed overtime and removed distinct

**Review by colegottdank — APPROVED** [review]

**Review by chitalian — APPROVED** [review]

**Pull request merged**